wrappers/pointcloud/pointcloud-stitching/doc/pointcloud-stitching-demo.md

```diff
@@ -0,0 +1,189 @@
+    If you reuse the files created for the gathering of images process, you can just remark the INFRARED streams using the number sign (#).</br>
+    The files should look as follows:
+
+    *912112073098.cfg:*</br>
+    ```
+    DEPTH,640,480,30,Z16,0
+    COLOR,640,360,30,RGB8,0
+    #INFRARED,640,480,30,Y8,1
+    ```
+
+    *831612073525.cfg:*</br>
+    ```
+    DEPTH,640,480,30,Z16,0
+    COLOR,640,360,30,RGB8,0
+    #INFRARED,640,480,30,Y8,1
+    ```
+
+2.	**A calibration file.**</br>
+    - The name of the file is unimportant as you specify it in `rs-pointcloud-stitching` command line.</br>
+    - It should contain 2 lines. One contains the calibration matrix between camera A and camera B. The other, the calibration matrix between camera B and the virtual camera that the app creates.</br>
+    - The transformation parameters are 9 rotation and then 3 translation parameters.
+
+    </br>
+    As an example, a calibration file describing 2 cameras with 15 degrees between them is given:</br>
+
+    *calibration_15.cfg*:
+    ```
+    912112073098, 831612073525, 0.8660254, 0,  -0.5,  0,1,0,      0.5, 0., 0.8660254, 0,0,0
+    831612073525, virtual_dev, 0.96592583, 0,  0.25881905,  0,1,0,      -0.25881905, 0., 0.96592583, 0,0,0
+    ```
+    **Notice:** "virtual_dev" is a key word.
+
+    Following the suggested calibration process, a calibration file was already created by export_calibratrion.m (C:\pc_stitching_ws\calibration_60m.cfg in this example).
+
+    *calibration_60m.cfg:*
+    ```
+    912112073098, 831612073525, 0.488316, -0.007477, -0.872635, 0.004309, 0.999972, -0.006157, 0.872656, -0.000754, 0.488334, 0.199384, 0.001205, -0.090225
+    831612073525, virtual_dev, 0.862643, 0.003806, 0.505799, -0.004022, 0.999992, -0.000666, -0.505797, -0.001460, 0.862651, -0.099692, -0.000602, 0.045112
+    ```
+
+    The first line describes the calibration from camera 912112073098 to 831612073525.</br>
+    The second line describes the calibration from 831612073525 to the virtual device.
+ 
+
+3.	**Defining the virtual device:**</br>
+rs-pointcloud-stitching requires a file defining the virtual device created by combining the other 2 devices.</br>
+It should contain wanted FOV and resolution for the virtual device.</br>
+For example:
+
+    *virtual_dev.cfg:*</br>
+    ```
+    depth_width=960
+    depth_height=320
+    depth_fov_x=120
+    depth_fov_y=60
+
+    color_width=720
+    color_height=240
+    color_fov_x=120
+    color_fov_y=40
+    ```
+    There is no hard limitation on the values given to the virtual device.
+    Some notes to bear in mind for appearance:
+    -	Areas in the virtual device's FOV that are not covered by actual devices create empty spaces in the image.
+    -	Try to maintain the same width-height ration for color and depth images. 

```

Comment:
1.ratio
2. I think this is more than recommendation. Otherwise the pixels will be rectangular and not square

Reply:
Yes, they will be rectangular but this is not a hard limitation on the demo. A user can choose whatever seems nicer to them. I played around a little to find a visually pleasing result. If I came up with a strict rule that should be observed I would enforce it in the code or better yet, allow the user to set only part of the list and make the program fill the rest.